subseas_flex_fcst/layout.py

```diff
@@ -228,7 +248,66 @@ def navbar_layout(phys_units):
                 ],
                 id="threshold_style"
             ),
-            dbc.Alert(
+            html.Div(
+                [
+                    help_layout(
+                        "Start Date",
+                        "start_date",
+                        "Start date choices",
+                    ),
+                ],
+                style={
+                    "color": "white",
+                    "position": "relative",
+                    "width": "105px",
+                    "display": "inline-block",
+                    "padding": "10px",
+                    "vertical-align": "top",
+                }
+            ),
+            html.Div(
+                [
+                    dcc.Dropdown(
+                        id="startDate",
+                        clearable=False,
+                        options = startDates,
+                        value=startDates[-1],
```

Comment:
In Ingrid's version, I reverted the order of the list so that he most recent Start date are at the top of the menu. Supposedly users will be interested in the most recent stuff, so it makes sense to do that. It's possibly a little less true with the type-in functionality to find possible options, as it makes their ordering less relevant.
Either way... no strong opinion on reverting order or not.

Reply:
I will make note of this, and can ask for feedback when I show the new changes.